# MAVLink mode information

std_msgs/Header header

int16 base_mode
int16 custom_mode
bool mission_start
